static/js/popover_menus.js

```diff
@@ -185,4 +202,137 @@ export function initialize() {
             compose_enter_sends_popover_displayed = false;
         },
     });
+
+    delegate("body", {
+        ...default_popover_props,
+        target: ".actions_hover .zulip-icon-ellipsis-v-solid",
+        placement: "bottom",
+        popperOptions: {
+            modifiers: [
+                {
+                    // The placement is set to bottom, but if that placement does not fit,
+                    // the opposite top placement will be used.
+                    name: "flip",
+                    options: {
+                        fallbackPlacements: ["top"],
+                    },
+                },
+            ],
+        },
```

Comment:
This works nicely and we can use it in others places as well.

Reply:
OK; I've not played with it UI-wise, but we could consider extracting that as a `POPPER_FLIP_CONFIGURATION` variable or something if we expect to use it widely.

Reply:
We would be using different `fallbackPlacements` for others, so, it would be better to just use it directly.

Reply:
Sounds good.